src/suricata.c

```diff
@@ -2242,8 +2242,10 @@ static int MayDaemonize(SCInstance *suri)
 static int InitSignalHandler(SCInstance *suri)
 {
     /* registering signals we use */
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
please split this commit. It does different things:
1. make PostConfLoadedSetup global
2. fix PostConfLoadedSetup using global &suricata instead of local suri ptr
3. change main
4. change signals.

Reply:
ok